packages/infra/lib/api-stack.ts

```diff
@@ -1494,6 +1504,37 @@ export class APIStack extends Stack {
     return fhirToMedicalRecordLambda;
   }
 
+  private setupFhirToCdaConverterLambda(ownProps: {
+    lambdaLayers: LambdaLayers;
+    vpc: ec2.IVpc;
+    medicalDocumentsBucket: s3.Bucket;
+    envType: EnvType;
+    sentryDsn: string | undefined;
+    alarmAction: SnsAction | undefined;
+  }): Lambda {
+    const { lambdaLayers, vpc, medicalDocumentsBucket, sentryDsn, alarmAction, envType } = ownProps;
+
+    const fhirToCdaConverterLambda = createLambda({
+      stack: this,
+      name: "FhirToCdaConverter",
+      runtime: lambda.Runtime.NODEJS_18_X,
+      entry: "fhir-to-cda-converter",
+      envType,
+      envVars: {
+        MEDICAL_DOCUMENTS_BUCKET_NAME: medicalDocumentsBucket.bucketName,
+        ...(sentryDsn ? { SENTRY_DSN: sentryDsn } : {}),
+      },
+      layers: [lambdaLayers.shared],
+      memory: 512,
```

Comment:
It would be good to run a couple tests to make sure this is enough, as files can get large. Or start with 1GB and add a ticket/task on calendar to check after a month.

Better to downgrade if not needed than failing, since Lambdas are quite cheap to run.